src/main/java/com/google/step/snippet/external/GeeksForGeeksClient.java

```diff
@@ -11,6 +11,8 @@
   private static final String DESC_TAG = "p";
   private static final String SNIPPET_CLASS = "code-block";
   private static final String CODE_CLASS = "code-container";
+  private static final String SOURCE_NAME = "GeeksForGeeks";
+  private static final String ICON_LINK = "https://geeksforgeeks.org/favicon.ico";
```

Comment:
Turns out my adblocker, uBlock Origin, blocks favicons used like this. Who knew? Oh well.